src/components/modals/qr.tsx

```diff
@@ -45,7 +82,8 @@ export const QRModal = ({onClose}: QRModalProps) => {
 
   const checkAddress = useCallback((address: string) => {
     if (utils.isAddress(address)) {
-      app.emit('address', address);
+      setIsOpen(true);
```

Comment:
not all cases should open selector
if we open scan qr from card we should go directly to transaction

Reply:
`app.emit('modal', {type: 'qr', showSelector: true});`